content/tracing/setup/python.md

```diff
@@ -73,9 +84,43 @@ ___
 | SQLite        | https://www.sqlite.org/                       | http://pypi.datadoghq.com/trace/docs/#sqlite        |
 {{% /table %}}
 
+## Example: Simple tracing
+
+We have a Flask Python application that when called on `/doc` returns **42**
```

Comment:
Should there be a period at the end of this sentence?

Reply:
👌